package.json

```diff
@@ -47,11 +47,10 @@
     "eslint": "^2.13.1",
     "eslint-config-loopback": "^2.0.0",
     "grunt": "^1.0.0",
-    "grunt-docular": "~0.1.2",
     "grunt-loopback-sdk-angular": "^1.2.0",
-    "mocha": "^2.4.5",
+    "mocha": "^5.2.0",
     "mysql": "^2.4.2",
     "read": "^1.0.5",
-    "supertest": "^1.2.0"
+    "supertest": "^3.0.0"
```

Comment:
The error might be caused by the upgrading of supertest.
@nabdelgadir you could try revert the version to `^1.2.0` to narrow down the failing reason. If tests pass, then we can upgrade the api to fit version 3.x